thirdparty_builtin/googletest/googletest/cmake/internal_utils.cmake

```diff
@@ -260,16 +260,22 @@ if (POLICY CMP0094)
   cmake_policy(SET CMP0094 NEW)
 endif()
 
-# Sets PYTHONINTERP_FOUND and PYTHON_EXECUTABLE.
-if ("${CMAKE_VERSION}" VERSION_LESS "3.12.0")
-  find_package(PythonInterp)
-  set(PYTHONINTERP_FOUND ${PYTHONINTERP_FOUND} CACHE INTERNAL "")
-  set(PYTHON_EXECUTABLE ${PYTHON_EXECUTABLE} CACHE INTERNAL "")
-else()
-  find_package(Python COMPONENTS Interpreter)
-  set(PYTHONINTERP_FOUND ${Python_Interpreter_FOUND} CACHE INTERNAL "")
-  set(PYTHON_EXECUTABLE ${Python_EXECUTABLE} CACHE INTERNAL "")
-endif()
+#!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
```

Comment:
Can you add a patch of this and put it in this directory `thirdparty_builtin/patches`? I can do it if you like.